tests/Rector/v9/v0/QueryLogicalOrAndLogicalAndToArrayParameterRector/Fixture/query_logical_and_with_array_but_additional_parameter.php.inc

```diff
@@ -33,7 +33,7 @@ class ProductRepositoryLogicalAndButArrayAndAdditionalParameter extends Reposito
         $query->matching($query->logicalAnd([
                 $query->lessThan('foo', 1),
                 $query->lessThan('bar', 1),
-            ]
+            ], 'foo'
```

Comment:
This test changes its output because of the new Query stub. What shall I do here?